fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1420,43 +1420,20 @@ public long saveAlterJob(DataOutputStream dos, long checksum) throws IOException
     }
 
     public long saveAlterJob(DataOutputStream dos, long checksum, JobType type) throws IOException {
-        Map<Long, AlterJob> alterJobs = null;
-        ConcurrentLinkedQueue<AlterJob> finishedOrCancelledAlterJobs = null;
         Map<Long, AlterJobV2> alterJobsV2 = Maps.newHashMap();
         if (type == JobType.ROLLUP) {
-            alterJobs = this.getRollupHandler().unprotectedGetAlterJobs();
```

Comment:
I think there are no AlterJob and SchemaJob in current starrocks?
Why not delete them directly?

Reply:
Old AlterJob has no cleanup mechanism, Therefore, it is possible to keep AlterJob in the image all the time, so it is better to go in two steps. The first step is to clear the image in 2.4, and it will be safer to delete the code in the next version about 2.5.